modules/apps/segments/segments-web/src/main/resources/META-INF/resources/segments_experience_selector.jsp

```diff
@@ -56,15 +67,17 @@ SegmentsExperienceSelectorDisplayContext segmentsExperienceSelectorDisplayContex
 							</p>
 						</div>
 
-						<c:if test='<%= segmentsExperiencesJSONObject.getBoolean("active") %>'>
-							<div class="autofit-col">
-								<span class="label label-success m-0">
-									<span class="label-item label-item-expand">
-										<liferay-ui:message key="active" />
-									</span>
+						<%
+						boolean active = segmentsExperiencesJSONObject.getBoolean("active");
+						%>
+
+						<div class="autofit-col">
+							<span class="label label-<%= active?"success":"secondary" %> bg-transparent m-0">
+								<span class="label-item label-item-expand">
+									<%= segmentsExperienceSelectorDisplayContext.getSegmentsExperienceStatusLabel(active) %>
```

Comment:
Maybe we can move active value inline as a parameter, like:

segmentsExperienceSelectorDisplayContext.getSegmentsExperienceStatusLabel(
segmentsExperiencesJSONObject.getBoolean("active"))